tools/stmo.md

```diff
@@ -1,5 +1,276 @@
+each visualization rendering the extracted data in some more useful
+format. There are many visualization types, including charts (line, bar, area,
+pie, etc.), counters, maps, pivot tables, and more. Each visualization type
+provides a set of configuration parameters that allow you to specify how to map
+from the raw query output to the desired visualization. Some visualization types
+make demands of the query output; a map visualization requires each row to contain
+a longitude value and a latitude value, for instance.
+
+#### Dashboards
+
+A dashboard is a collection of visualizations, combined into a single visual
+presentation for convenient perusal. A dashboard is decoupled from any
+particular queries. While it is possible to include multiple visualizations
+from a single query in one dashboard, it is not required; users can add any
+visualizations they can access to the dashboards they create.
+
+Data Sources
+------------
+
+SQL provides the ability to extract and manipulate the data, but you won't get
+very far without having some familiarity with what data is actually available,
+and how that data is structured. Each query in STMO is associated with exactly
+one data source, and you have to know ahead of time which data source contains
+the information that you need. One of the most commonly used data sources is
+called *Athena* (referring to Amazon's [Athena](https://aws.amazon.com/athena/)
+query service, on which it is built), which contains most of the data that is
+obtained from telemetry pings received from Firefox clients. The *Athena*
+source is slowly replacing the *Presto* data source. *Presto* contains all of
+the data that's exposed via *Athena* and more, but returns query results much
+more slowly.
+
+Other available data sources include *Crash-DB*, *Tiles*, *Sync Stats*, *Push*,
+*Test Pilot*, *ATMO*, and even a *re:dash metadata* which connects to STMO's
+own redash database. It is beyond the scope of this document to list and
+describe all of the data sources, but if you have questions about particular
+data sets, or would like to know if specific data is or can be made available
+in STMO, please inquire in the #datapipeline or #datatools channels on
+irc.mozilla.org.
+
+Creating an Example Dashboard
+-----------------------------
+
+The rest of this document will take you through the process of creating a
+simple dashboard using STMO.
+
+#### Creating A Query
+
+First we must create a query, which can be initiated by clicking on the 'New
+Query' button near the top left of the page. This will bring you to the query
+editing page:
+
+![New Query Page](/../assets/STMO_screenshots/new_query.png "New Query page")
+
+The first query will be counting the number of client ids that we have coming
+from each country, for the top N countries. There is a "client_count" data set
+that is generated from Firefox telemetry pings, so first we check to see if
+that data is available in the *Athena* data source by typing "client_count"
```

Comment:
It may be helpful to break these steps into bullet points. It would be useful to be able to visually identify the splits between the tasks, e.g.:
* Check if the datasets in Athena
* Verify it's in Presto
* Take a look at the available columns